xclim/testing/tests/test_cli.py

```diff
@@ -292,3 +292,19 @@ def test_bad_usage(tas_series, tmp_path):
         assert "distributed scheduler is not installed" in results.output
     else:
         assert "'--dask-maxmem' must be given" in results.output
+
+
+def test_release_notes():
+    runner = CliRunner()
+    for method, pattern in {"-r": "`GH/", "-m": "[GH/"}:
```

Comment:
```suggestion
    for method, pattern in [("-r", "`GH/"), ("-m", "[GH/")]:
```
I think this is it! Iterating over a dictionary directly yields the keys. Here, you have 2 args so the len-2 strings are split. The first case was method='-' and patter='r'. 

We didn't see the error because the cli did run. It failed and outputted some error messages, but none of the asserts failed. That's evident for the 3 firsts, and the last one was 'assert 'r' in results.output', which had high probabilities of passing ;)!

Reply:
In fact, I would suggest using `pytest.mark.parametrize` here. I don't think your supposed to use the same runner instance twice? Not sure though.